src/main/scala/scalaz/ziokeeper/VectorClock.scala

```diff
@@ -0,0 +1,63 @@
+package scalaz.ziokeeper
+
+import scala.annotation.tailrec
+import VectorClock._
+case class VectorClock[A](value: Map[A, Long]) extends AnyVal {
+  def time(mid: A): Long = value.getOrElse(mid, 0L)
+
+  def increment(mid: A, value: Long): VectorClock[A] =
```

Comment:
This vector clock will only ever be incremented by `1`. We will maybe need to "set" certain components for other nodes.

Reply:
changed to `def increament(mid: A)`. we don't need set because merge should be enough.